values.yaml

```
@@ -502,10 +502,10 @@ concourse:
       ##
       useCaCert: false
 
-      ## Vault authentication backend, leave this blank if using an initial periodic token.
+      ## Vault authentication backend
       ## Currently supported backends: token, approle, cert.
-      ##
-      authBackend: ""
+      ## Use token if using an initial periodic token.
+      authBackend: token
```

Comment update is great. Only thing I think should change is the value.

```suggestion
      authBackend:
```
I think we should not set the default to token to match what the Concourse binary does, which is not setting a default for this setting. This is what we've been doing when PR's like this come up.

From the bottom of the [README](https://github.com/concourse/concourse-chart#developing):

> When adding a new Concourse flag, don't assign a default value in the values.yml that mirrors a default set by the Concourse binary.

We haven't spent the time to retroactively do this to all the existing vars for the chart, only as PR's come up or if we're modifying relevant sections.

We should also wrap the `CONCOURSE_VAULT_AUTH_BACKEND` env in an `if` so it doesn't render when the value is blank (inside `web-deployment.yaml`).

What do you think about making those changes?